src/Acuminator/Acuminator.Vsix/Utils/SuppressionFileSystemService.cs

```diff
@@ -28,5 +40,16 @@ public string GetFileName(string path)
 
 			return Path.GetFileNameWithoutExtension(path);
 		}
+
+		public ISuppressionFileWatcherService CreateWatcher(string path)
+		{
+			var directory = Path.GetDirectoryName(path);
+			var file = Path.GetFileName(path);
+			var watcher = new FileSystemWatcher(directory, file);
+
+			watcher.NotifyFilter = NotifyFilters.LastWrite;
```

Comment:
@metallmonster why only specify LastWrite? I'd expect almost everythyng watched here - Security, size changes, file name changes, etc. Imho only LastAccess and perhaps CreationTime should not be watched

Reply:
@SENya1990 good point